Packs/AWS-IAMIdentityCenter/Integrations/AWSIAMIdentityCenter/AWSIAMIdentityCenter.yml

```diff
@@ -1,5 +1,5 @@
 commonfields:
-  id: AWS - IAM Identity Center
+  id: AWS - IAM Identity Center - TEST
   version: -1
 category: IT Services
 sectionOrder:
```

Comment:
Why do we need the collect section? 
If no good reason please remove it from all relevant places